syntax = "proto3";

package message; //理解为命名空间

service VerifyService {
  rpc GetVerifyCode (GetVerifyReq) returns (GetVerifyRsp) {}
}

message GetVerifyReq {		//这里的message是protobuf里的一种数据类型，不是指上面的package
  string email = 1;
}

message GetVerifyRsp {
  int32 error = 1;    //gRPC服务结果
  string email = 2;
  string code = 3;
}